pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/JdbiQuestion.sql.stg

```diff
@@ -89,6 +89,13 @@ select qt.question_type_code as question_type,
          where dqfo.date_question_id = dq.question_id
        ) as date_fields,
 
+       (select group_concat(mt.mime_type separator ',')
```

Comment:
Nice! But I think [mysql server has a limit](https://dev.mysql.com/doc/refman/5.7/en/aggregate-functions.html#function_group-concat) on the string length returned from `group_concat`.

> The result is truncated to the maximum length that is given by the `group_concat_max_len` system variable, which has a default value of 1024.

`group_concat` works well for something like the date fields since there's only 3 of those (month, day, year), but the mime types are bigger strings in general. Let's assume an average of 20 characters per mime type, we can concat about 50 of those, so maybe this should be fine?

Reply:
Is it really possible a lot of mime-types in one File Question?
Actually it would be much more convenient to group mime-types (may be to define enum fo it) and then assign to FileQiestions the groups instead of mime types. 
Like the following groups: IMAGE ("image/gif","image/jpeg"...), VIDEO ("video/mpeg", "video/ogg",...), etc.
I mean that in the study builder it would be much more convenient to operate with such groups of mime types than each time to remember which mime types possible for image or video or text formats..
And when returning to frontend a group would be translated to a real list of mime types..

Reply:
> Let's assume an average of 20 characters per mime type, we can concat about 50 of those, so maybe this should be fine?
I think it could be enough.

But I can provide an alternative way to read mime types during questions data reading: read it by a separate query
in this method (in class JdbiQuestion.java):
```
    Stream<QuestionDto> findQuestionDtosByIds(
            @BindList(value = "questionIds", onEmpty = EmptyHandling.NULL) Iterable<Long> questionIds);
```
If it is ok then I'll do this change.

Reply:
I think typical use-case we'll see is allowing PDFs, or images, so just a handful of mime types. So I think we should be fine for now.